javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,85 @@
+let yourName = "";
+const toDosList = [];
+function getReply(command) {
+
+    if (command === "Hello my name is Benjamin" && !yourName) {
+        yourName += "Benjamin";
+        return "Nice to meet you " + yourName;
+    } else if (command === "Hello my name is Benjamin" && yourName) {
+        return "Yeah I know.. Nice to see you again by the way";
+    }
+
+    if (command === "What is my name" && !yourName) {
+        return "I don´t know, what´s your name?";
+    } else if (command === "What is my name" && yourName) {
+        return "Your name is " + yourName;
+    }
+
+    if (command === "Add fishing to my todo") {
+        toDosList.push("Fishing");
+        return "Fishing added to my todo"
+    }
+
+    if (command === "Add singing in the shower to my todo") {
+        toDosList.push("Singing in the shower");
+        return "Singing in the shower added to my todo";
+    }
+
+    if (command === "Remove fishing from my todo") {
+        let fishingIndex = toDosList.indexOf("Fishing");
+        if (fishingIndex < 0) {
+            return "I couldn´t find 'fishing' in your todo list";
+        } else {
+            toDosList.splice(fishingIndex, 1);
+            return "Removed fishing from my todo";
+
+        }
+    }
+
+    if (command === "What is on my todo?") {
+        let toDosString = toDosList.join("\n")
+        return "You have " + toDosList.length + " on your todos:\n" + toDosString;
+    }
+
+    if (command === "What day is it today?") {
+        let todaysDate = new Date();
+        let options = {
+            dateStyle: "long",
+            /*year: "numeric",
+            month: "long",
+            day: "numeric",
+*/
```

Comment:
Git provides a complete record of our files so you can safely delete commented code. We can always get it back with Git if we need to.